src/main/resources/standardTestCases/bad_docker_name.test

```diff
@@ -0,0 +1,7 @@
+name: bad_docker_name
+testFormat: workflowfailure
+
+files {
+  wdl: bad_docker_name/bad_docker_name.wdl
+  inputs: hello/hello.inputs
+}
```

Comment:
Would it be helpful to add the check for the type of error thats returned in Metadata?